EIPS/eip-7805.md

```diff
@@ -85,31 +85,46 @@ When validators receive ILs from the P2P network, they perform a series of valid
 
 ### Execution Layer
 
-On the execution layer, the block validity conditions are extended such that, after all of the transactions in the block have been executed, we attempt to execute each valid transaction from ILs that was not present in the block.
-If one of those transactions executes successfully, then the block is invalid.
+On the execution layer, an additional check is introduced for new payloads. After all of the transactions in the payload have been executed, we check whether any transaction from ILs, that is not already present in the payload, could be validly included (i.e. nonce and balance checks pass). If that is the case for any transaction, then an error is returned to the CL. Although the block is valid, the CL will not attest to it.
```

Comment:
Besides `nonce` and `balance` check other checks have to be passed as well. For instance: does it allow to pay the `baseFee`? (Other tx validity checks which are independent of the block environment (e.g. `baseFee` which depends on the current block) such as `tx.gaslimit` is sufficient (pays intrinsic fee) should be checked as well. (This is a note: the balance/nonce should be the most prominent checks but obviously other params have to be checked as well -> this is ok to be motivated later, but a mention of this here would be more clear)